script.js

```diff
@@ -0,0 +1,30 @@
+const button = document.getElementById("button");
+const input = document.getElementById("input");
+const form = document.getElementById("form");
+const table = document.getElementById("table") 
+
+let number = "";
+
+form.addEventListener("submit",(e)=>{
+    e.preventDefault();
+})
```

Comment:
Isto é apenas preferências, mas quando as funções são assim pequenas eu gosto de meter tudo na mesma linha:
```suggestion
form.addEventListener("submit", e => e.preventDefault());
```

Reply:
E se quiseres até podes fazer assim:
```suggestion
document.getElementById("form").addEventListener("submit", e => e.preventDefault());
```